pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -233,6 +234,64 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 	// Determine if the gGRPC auth is enabled
 	config.GRPCAuthEnabled = helpers.GRPCAuthEnabled(clusterManager)
 
+	// Get ClusterProxy related configuration from cluster-manager.
```

Comment:
@qiujian16 Please take a review if this is the desired way of getting cluster-proxy related cofniguration from clustermanager.

My questions are:

1. clusterproxy configuration is determinted by `registrationConfiguration` and `serverConfiguration`. 

2. We have to range the `endpointsExposure` and compare `usage` field to find the cofniguration for cluster proxy servers.

Is it possible we can define a `ProxyConfiguration` struct in ClusterManager like Klusterlet did?